encord/http/v2/api_client.py

```diff
@@ -4,6 +4,7 @@
 from urllib.parse import urljoin
 
 import requests
+from pydantic import BaseModel
```

Comment:
Checked that this is Pydantic V1/ V2 agnostic?
Should probably include a comment describing this behaviour given it can look opaque out of context

Reply:
yes, the same import works for both cases